client/main-layout.tsx

```diff
@@ -341,14 +341,14 @@ export function MainLayout() {
           icon={<DownloadIcon />}
           label='Download'
           onClick={() => dispatch(openDialog(DialogType.Download))}
-          hotkey={ALT_O}
+          hotkeys={[ALT_O]}
         />,
         <ActivityButton
           key='ladder'
           icon={<LadderIcon />}
           label='Ladder'
           onClick={() => navigateToLadder()}
-          hotkey={ALT_D}
+          hotkeys={[ALT_D]}
```

Comment:
If you do this you're creating a new array every time and thus the whole reason for pulling these out (that they memoize properly) is lost